client/components/sites-dropdown/test/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import useFilesystemMocks from 'test/helpers/use-filesystem-mocks';
+
+describe( 'index', function() {
+	useFakeDom();
```

Comment:
Enzyme's mount API requires a DOM, while shallow does not. So if the component/required libs aren't calling out directly to items available on the browser (in this case window) we don't need to set one up.